understanding/20/use-of-color.html

```diff
@@ -81,7 +81,29 @@ <h2>Intent of Use of Color</h2>
          </p>
 
       </div>
-      
+
+      <div class="note">
+
+         <p>Under certain conditions, most user agents provide users with a color-only cue that a link has been visited. However, due to several technical constraints, authors should not be held accountable for this use of color, and color-only indications of visited links should not result in a failure of Use of Color (or a number of other criteria). The technical constraints are as follows:
```

Comment:
possibly instead of the `authors should not be held accountable for ...` angle, it may be a bit more "neutral" (rather than emphasising / suggesting WCAG is a way to assign culpability) to rephrase this as `authors have limited control over ...`

Reply:
Made some changes. LMK if they suffice

Reply:
lovely, much better